client/game-page/src/main/kotlin/com/bytelegend/client/app/ui/CoinCountWidget.kt

```diff
@@ -40,6 +45,21 @@ class CoinCountWidget : AbstractIncrementAnimatableWidget<CoinCountWidgetProps,
             +game.heroPlayer.coin.toString()
         }
         renderIcon()
+        attrs.onClickFunction = {
```

Comment:
DRY.

You have two options:

- Use `game.modalController.show(i("UnfinishedText2"), i("UnfinishedTitle"))`
- Use 

```
game.modalController.show {
    child(HistoryModal::class) {}
}
```

and extract the logic to `HistoryModal`. Option 2 is preferred because we'll add more logic to it soon.